src/index.html

```diff
@@ -17,16 +17,21 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form data-qa="big">
+      <input
+        class="icon search-bar"
+        type="search"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
+    <form data-qa="small">
+      <input
+        class="icon search-bar"
+        type="search"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
```

Comment:
There are two forms with identical input fields. If they serve different purposes, consider adding distinguishing attributes or IDs. If not, you might want to remove one to avoid redundancy.